bot/utilities/Queue.ts

```diff
@@ -48,4 +50,45 @@ export class Queue {
     this.properties = { id, ownerId, channelId, startTime, status };
     this.entries = [];
   }
+
+  get length(): number {
+    return this.entries.length;
+  }
+  
+  checkQueue(context: TurnContext): boolean {
+    if (this.entries.find((student) => student.userId == context.activity.from.id) != undefined) {
+      return true;
+    }
+    return false;
+  }
+
+  enqueueStudent(context: TurnContext): void {
+    const studentToAdd: QueueEntry = {
+      id: Math.random() * 1000,
+      userId: context.activity.from.id,
```

Comment:
Same situation here: I think it would be cleaner to pass the `userId` property directly rather than the entire context object.

Reply:
Sounds good!